TypeCobol/Compiler/CodeElements/StatementElement.cs

```diff
@@ -127,6 +127,9 @@ public enum StatementType
         UseStatement,
         WriteStatement,
         XmlGenerateStatement,
-        XmlParseStatement
+        XmlParseStatement,
+
+        //
+        LastStatementType,
```

Comment:
I see that you used those 'last enum element' to split enums across two enum types. It works but it also introduces a fake element in the enum... I don't know how to solve this.